Extensions/Physics2Behavior/JsExtension.js

```diff
@@ -1076,8 +1076,21 @@ module.exports = {
       .setFunctionName('setLinearVelocityX')
       .setManipulatedType('number')
       .setGetter('getLinearVelocityX');
-
-    aut
+	
+	aut
```

Comment:
Remove one indentation live for "aut" (i.e: two space, like the other expressions/instructions)